mozilla/bedrock
# [fix bug 1436737] Remove blog thumbnail links from global nav side drawer

## Description
- Removes blog thumbnail links from the global nav side drawer which have not been updated in over a year.
- Removing this old content reduces the number of image requests for every page load.

## Issue / Bugzilla link
https://bugzilla.mozilla.org/show_bug.cgi?id=1436737

## Testing
- Ensure no regressions
- Make sure I didn't miss anything?
